src/corehost/packaging/dir.props

```diff
@@ -23,7 +23,7 @@
 
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23712\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23931\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
Does your UpdateDependencies script have a way to update non-project.json files? I actually did a fair amount of hacking in the meta-package building to try and make sure my dependencies were listed in a project.json.